ios/Video/RCTVideo.m

```diff
@@ -1274,27 +1274,23 @@ - (void)setFilter:(NSString *)filterName {
 
         CIFilter *filter = [CIFilter filterWithName:filterName];
 
-        _playerItem.videoComposition = [AVVideoComposition
-                videoCompositionWithAsset:asset
-             applyingCIFiltersWithHandler:^(AVAsynchronousCIImageFilteringRequest *_Nonnull request) {
+        if (filter != nil) {
```

Comment:
Actually, need to test for an empty filter name too. Having set no filter, I get "" here, not nil